### DISCUSSION TITLE
part-ii/2017-113/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#33] [Question comments]
Part II 2017 Principles of Quantum Mechanics: Paper 2, Section II, C 

https://questions.tripos.org/part-ii/2017-113/

**Comment by silkca** [comment]
$(\mathbf{n} \cdot \boldsymbol{\sigma})^{2}=1$ is really a very strong statement - using $\mathbf{n}=\mathbf{e_i}$ and $\mathbf{n}=\frac{\mathbf{e_i}+\mathbf{e_j}}{\sqrt(2)}$ provides plenty of useful information, however the manipulation is still a bit of an algebra bash from there. There may be a cleverer choice of $\mathbf{n}$.

I'm really quite bad at the interpretations, so I just said "this is the spin operator acting on a spin $\frac{1}{2}$ particle".

The key to the second half is to notice that when $\mathbf{n} \cdot \boldsymbol{\sigma}$ has eigenvalues $\pm1$, $P_{\pm}$ projects on to the corresponding eigenspaces. The question has you prove almost all the relations which determine this, but there are a few others which aren't difficult.

For the last part, the only interesting insights are that if $(\mathbf{m}\cdot\boldsymbol{\sigma})|\chi\rangle=|\chi\rangle$, then $(\mathbf{n}\cdot\boldsymbol{\sigma})(\mathbf{m}\cdot\boldsymbol{\sigma})|\chi\rangle=(\mathbf{n}\cdot\boldsymbol{\sigma})|\chi\rangle$, and $i(\mathbf{n}\times\mathbf{m}) \cdot \boldsymbol{\sigma}|\chi\rangle=0$ as $\mathbf{m}$ and $\mathbf{n}\times\mathbf{m}$ are perpendicular.
